### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by GuidoS**

### CODE REVIEW

**Comment by GuidoS** [comment]
@calvinmetcalf

File: bin.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+var argv = require('yargs')
+  .usage('Usage: $0 config_file -sql')
+  .demand(1)
+  .alias('q', 'query')
+  .describe('q', 'sql query')
+  .alias('u', 'unique')
+  .describe('u', 'unique field, non-nullable')
+  .argv;
+
+var config = JSON.parse(fs.readFileSync(argv._[0], 'utf8'));
```
Comment (calvinmetcalf) [inline bin.js]: we may want to check that this is actually present

File: bin.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var fromMssql = require('./');
+var fs = require('fs');
+var argv = require('yargs')
+  .usage('Usage: $0 config_file -sql')
```
Comment (calvinmetcalf) [inline bin.js]: what is the `-sql` refering to?

File: bin.js (hunk 1 above)
Comment (GuidoS) [inline bin.js]: ok

File: bin.js (hunk 2 above)
Comment (GuidoS) [inline bin.js]: older code being referenced here, will remove and update

**Pull request closed without merging**